__tests__/app.test.js

```diff
@@ -29,24 +30,65 @@ describe("GET /api/topics", () => {
         });
       });
   });
-  test("404: responds with not found for wrong endpoint", () => {
+  // test("404: responds with not found for wrong endpoint", () => {
+  //   return supertest(app)
+  //     .get("/api/something")
+  //     .expect(404)
+  //     .then((response) => {
+  //       expect(response.body).toEqual({ error: "Not found" });
+  //       expect(response.statusCode).toBe(404);
+  //     });
+  // });
+});
+
+describe("/api", () => {
+  test("200 - responds with json file containing list of endpoints", () => {
     return supertest(app)
-      .get("/api/something")
-      .expect(404)
+      .get("/api")
+      .expect(200)
       .then((response) => {
-        expect(response.body).toEqual({ error: "Not found" });
-        expect(response.statusCode).toBe(404);
+        expect(jsonfile);
```

Comment:
expect jsonfile to what? This test is not asserting anything and will be passing as a false positive.